yb-voyager/cmd/liveMigrationReportCommand.go

```diff
@@ -0,0 +1,264 @@
+		schemaName := strings.Split(table, ".")[0]
+		tableExportStatus := status.GetTableStatusByTableName(tableName, schemaName)
+		if tableExportStatus == nil {
+			tableExportStatus = &dbzm.TableExportStatus{
+				TableName:                tableName,
+				SchemaName:               schemaName,
+				ExportedRowCountSnapshot: 0,
+				FileName:                 "",
+			}
+		}
+		row.SnapshotRowCount = tableExportStatus.ExportedRowCountSnapshot
+		source = *msr.SourceDBConf
+		sourceSchemaCount := len(strings.Split(source.Schema, "|"))
+		row.TableName = table
+		if sourceSchemaCount <= 1 {
+			schemaName = ""
+			row.TableName = tableName
+		}
+		row.DBType = "Source"
+		err := updateRowForOutCounts(&row, tableName, schemaName) //source OUT counts
+		if err != nil {
+			utils.ErrExit("error while getting exported events counts for source DB: %w\n", err)
+		}
+		if fBEnabled {
+			err = updateRowForInCounts(&row, tableName, schemaName, msr.SourceDBAsTargetConf) //fall back IN counts
+			if err != nil {
+				utils.ErrExit("error while getting imported events for source DB in case of fall-back: %w\n", err)
+			}
+		}
+
+		reportTable.AddRow(row.TableName, row.DBType, row.SnapshotRowCount, row.InsertsOut, row.UpdatesOut, row.DeletesOut, row.InsertsIn, row.UpdatesIn, row.DeletesIn, getFinalRowCount(row))
+		row = rowData{}
+		row.TableName = ""
+		row.DBType = "Target"
+		err = updateRowForInCounts(&row, tableName, schemaName, msr.TargetDBConf) //target IN counts
+		if err != nil {
+			utils.ErrExit("error while getting imported events for target DB: %w\n", err)
+		}
+		if fFEnabled || fBEnabled {
+			err = updateRowForOutCounts(&row, tableName, schemaName) // target OUT counts
+			if err != nil {
+				utils.ErrExit("error while getting exported events for target DB: %w\n", err)
+			}
+		}
+		reportTable.AddRow(row.TableName, row.DBType, row.SnapshotRowCount, row.InsertsOut, row.UpdatesOut, row.DeletesOut, row.InsertsIn, row.UpdatesIn, row.DeletesIn, getFinalRowCount(row))
+		if fFEnabled {
+			row = rowData{}
+			row.TableName = ""
+			row.DBType = "Fall Forward"
+			err = updateRowForInCounts(&row, tableName, schemaName, msr.FallForwardDBConf) //fall forward IN counts
+			if err != nil {
+				utils.ErrExit("error while getting imported events for fall-forward DB: %w\n", err)
+			}
+			reportTable.AddRow(row.TableName, row.DBType, row.SnapshotRowCount, row.InsertsOut, row.UpdatesOut, row.DeletesOut, row.InsertsIn, row.UpdatesIn, row.DeletesIn, getFinalRowCount(row))
+		}
+	}
+	if len(tableList) > 0 {
+		fmt.Print("\n")
+		fmt.Println(reportTable)
+		fmt.Print("\n")
+	}
+
+}
+
+func updateRowForInCounts(row *rowData, tableName string, schemaName string, targetConf *tgtdb.TargetConf) error {
+	switch row.DBType {
+	case "Target":
+		importerRole = TARGET_DB_IMPORTER_ROLE
+	case "Fall Forward":
+		importerRole = FF_DB_IMPORTER_ROLE
+	case "Source":
+		importerRole = FB_DB_IMPORTER_ROLE
+	}
+	//reinitialise targetDB
+	tconf = *targetConf
```

Comment:
Will this overwrite the password set in the `tconf`?

Reply:
No, we are first taking the password set in `tconf` to set it back in msr.TargetConf in the starting after getting the password and then directly using that later.
```
if streamChanges {
  getTargetPassword(cmd)
  migrationStatus.TargetDBConf.Password = tconf.Password
  if migrationStatus.FallForwardEnabled {
  getFallForwardDBPassword(cmd)
  migrationStatus.FallForwardDBConf.Password = tconf.Password
  }
  if migrationStatus.FallbackEnabled {
  getSourceDBPassword(cmd)
  migrationStatus.SourceDBAsTargetConf.Password = source.Password
}
```